Cargo.toml

```diff
@@ -21,6 +21,7 @@ default = []
 docs = []
 
 [dependencies]
+ast_view = { version = "0.29.3", package = "dprint-swc-ecma-ast-view" }
```

Comment:
The name `dprint_swc_ecma_ast_view` is way too long. Is this ok to do instead?

Reply:
Fantastic